src/backend/distributed/commands/vacuum.c

```diff
@@ -296,10 +300,12 @@ DeparseVacuumStmtPrefix(CitusVacuumParams vacuumParams)
 {
 	int vacuumFlags = vacuumParams.options;
 	StringInfo vacuumPrefix = makeStringInfo();
+	bool is_vacuum = false;
```

Comment:
having an additional variable to track is_vacuum doesn't feel very intuitive given in all other cases we rely on vacuumFlags.

Can we do the same when is_vacuum is used such that we check vacuumFlags & VACOPT_VACUUM? If you prefer, you can also define a macro for that

Reply:
Let me try that one